app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::CommentsController < ApplicationController
+  skip_before_action :authenticate_user!
+  protect_from_forgery with: :null_session
+
+  def index
+    @post = Post.find(params[:post_id])
+    @comments = @post.comments
+
+    if @comments
+      render json: { status: 'Success', message: 'Comment fetched successfully', data: @comments }, status: :ok
+    else
+      render json: { status: 'Error', message: 'No comments found!', data: @comments.errors },
+             status: :bad_request
+    end
+  end
```

Comment:
- [ ] When I created posts using the frontend app, the API endpoint for returning posts for a user is not returning JSON data at the moment. Kindly work on this so it returns JSON data as expected. 👍🏽
- [ ] The same applies to the other endpoints 👍🏽